src/main/business/applicationFacade.js

```diff
@@ -283,6 +283,7 @@ class ApplicationFacade{
      * @throws {Error} - if there is no user logged in data
      */
     async viewMyTasks(data){
+        validateParameters(data,{})
         const user = await this.userController.getUser(data.callingUser.username);
         return this.taskController.getTasksOf(user.getUsername());
```

Comment:
Wouldn't it be easier to just call the `taskController` with `data` and extract the username from `callingUser` as part of the logic of `getTaskOf`?

Reply:
Nope there isnt...